website/content/docs/v0.9/Guides/troubleshooting-control-plane.md

```diff
@@ -0,0 +1,440 @@
+Single control plane node clusters are still used as test clusters, in edge deployments, but it should be noted that this setup is not HA.
+
+Number of control plane should be odd (1, 3, 5, ...), as with even number of nodes, etcd quorum doesn't tolerate
+failures correctly: e.g. with 2 control plane nodes quorum is 2, so failure of any node breaks quorum, so this
+setup is almost equivalent to single control plane node cluster.
+
+With three control plane nodes cluster can tolerate a failure of any single control plane node.
+With five control plane nodes cluster can tolerate failure of any two control plane nodes.
+
+### What is control plane endpoint
+
+Kubernetes requires having a control plane endpoint which points to any healthy API server running on a control plane node.
+Control plane endpoint is specified as URL like `https://endpoint:6443/`.
+At any point in time, even during failures control plane endpoint should point to a healthy API server instance.
+As `kube-apiserver` runs with host network, control plane endpoint should point to one of the control plane node IPs: `node1:6443`, `node2:6443`, ...
+
+For single control plane node clusters, control plane endpoint might be `https://IP:6443/` or `https://DNS:6443/`, where `IP` is the IP of the control plane node and `DNS` points to `IP`.
+DNS form of the endpoint allows to change the IP address of the control plane if that IP changes over time.
+
+For HA clusters, control plane can be implemented as:
+
+* TCP L7 loadbalancer with active health checks against port 6443
+* round-robin DNS with active health checks against port 6443
+* BGP anycast IP with health checks
+* virtual shared L2 IP
+<!-- TODO link to the guide -->
+
+It is critical that control plane endpoint works correctly during cluster bootstrap phase, as nodes discover
+each other using control plane endpoint.
+
+### kubelet is not running on control plane node
+
+Service `kubelet` should be running on control plane node as soon as networking is configured:
+
+```bash
+$ talosctl -n IP service kubelet
+NODE     172.20.0.2
+ID       kubelet
+STATE    Running
+HEALTH   OK
+EVENTS   [Running]: Health check successful (2m54s ago)
+         [Running]: Health check failed: Get "http://127.0.0.1:10248/healthz": dial tcp 127.0.0.1:10248: connect: connection refused (3m4s ago)
+         [Running]: Started task kubelet (PID 2334) for container kubelet (3m6s ago)
+         [Preparing]: Creating service runner (3m6s ago)
+         [Preparing]: Running pre state (3m15s ago)
+         [Waiting]: Waiting for service "timed" to be "up" (3m15s ago)
+         [Waiting]: Waiting for service "cri" to be "up", service "timed" to be "up" (3m16s ago)
+         [Waiting]: Waiting for service "cri" to be "up", service "networkd" to be "up", service "timed" to be "up" (3m18s ago)
+```
+
+If `kubelet` is not running, it might be caused by wrong configuration, check `kubelet` logs
+with `talosctl logs`:
+
+```bash
+$ talosctl -n IP logs kubelet
```

Comment:
```suggestion
$ talosctl -n <IP> logs kubelet
```

Reply:
We should probably surround all of these IPs with the brackets to make it clear the user needs to substitute something.